src/types/Auth.ts

```diff
@@ -0,0 +1 @@
+export type AuthType = 'authtoken' | 'password';
```

Comment:
Just to double check , these terms are standard vocabulary in NPM? Or is this an Azure DevOs concept?

Reply:
This is npm-specific vocabulary (`_authToken` vs `_password`) as `token` is only used for npm commands. I'm open to more "standardized" verbiage but couldn't think of anything that wouldn't also obfuscate the npm meaning.

Reply:
Sounds good! Just was curious where these terms came from